modules/apps/commerce/commerce-pricing-web/src/main/java/com/liferay/commerce/pricing/web/internal/portlet/action/EditCommercePriceEntryMVCActionCommand.java

```diff
@@ -222,16 +217,36 @@ protected CommercePriceEntry updateCommercePriceEntry(
 		boolean neverExpire = ParamUtil.getBoolean(
 			actionRequest, "neverExpire");
 
-		ServiceContext serviceContext = ServiceContextFactory.getInstance(
-			CommercePriceEntry.class.getName(), actionRequest);
+		boolean overrideDiscount = ParamUtil.getBoolean(
```

Comment:
move to 2nd position to sort variable as used

Reply:
Done